specification/keyvault/resource-manager/readme.nodejs.md

```diff
@@ -0,0 +1,13 @@
+## Node
+
+These settings apply only when `--nodejs` is specified on the command line.
+Please also specify `--node-sdks-folder=<path to the root directory of your azure-sdk-for-node clone>`.
+
+``` yaml $(nodejs)
+nodejs:
+  azure-arm: true
+  license-header: MICROSOFT_MIT_NO_VERSION
+  payload-flattening-threshold: 2
```

Comment:
I do not use this setting while generating the node sdk. Here is the output when I generate this locally from the node sdk gulp file.js

```
>>>>>>>>>>>>>>>>>>>Start: "azure-keyvault" >>>>>>>>>>>>>>>>>>>>>>>>>
Executing command:
------------------------------------------------------------
autorest --output-folder=/Users/amarz/sdk/azure-sdk-for-node/lib/services/keyvault/lib --package-name=azure-keyvault --nodejs --license-header=MICROSOFT_MIT_NO_VERSION  --azure-arm   https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/keyvault/data-plane/readme.md 
------------------------------------------------------------
Output:
AutoRest code generation utility [version: 2.0.4245; node: v8.9.1]
(C) 2018 Microsoft Corporation.
https://aka.ms/autorest
   Loading AutoRest core      '/private/var/folders/23/8g7f__gs1v39mrr49b8pc8pc0000gn/T/.autorest/@microsoft.azure_autorest-core@2.0.4245/node_modules/@microsoft.azure/autorest-core/dist' (2.0.4245)
   Installing AutoRest extension '@microsoft.azure/autorest.nodejs' (~2.1.25)
   Installing AutoRest extension '@microsoft.azure/autorest.modeler' (2.3.44)

>>>>>>>>>>>>>>>>>>>>>End: "azure-keyvault" >>>>>>>>>>>>>>>>>>>>>>>>>


>>>>>>>>>>>>>>>>>>>Start: "azure-arm-keyvault" >>>>>>>>>>>>>>>>>>>>>>>>>
Executing command:
------------------------------------------------------------
autorest --output-folder=/Users/amarz/sdk/azure-sdk-for-node/lib/services/keyVaultManagement/lib --package-name=azure-arm-keyvault --nodejs --license-header=MICROSOFT_MIT_NO_VERSION  --azure-arm   https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/keyvault/resource-manager/readme.md 
------------------------------------------------------------
Output:
AutoRest code generation utility [version: 2.0.4245; node: v8.9.1]
(C) 2018 Microsoft Corporation.
https://aka.ms/autorest
   Loading AutoRest core      '/private/var/folders/23/8g7f__gs1v39mrr49b8pc8pc0000gn/T/.autorest/@microsoft.azure_autorest-core@2.0.4245/node_modules/@microsoft.azure/autorest-core/dist' (2.0.4245)
   Loading AutoRest extension '@microsoft.azure/autorest.nodejs' (~2.1.25->2.1.28)
   Loading AutoRest extension '@microsoft.azure/autorest.modeler' (2.3.44->2.3.44)

>>>>>>>>>>>>>>>>>>>>>End: "azure-arm-keyvault" >>>>>>>>>>>>>>>>>>>>>>>>>
```